serialization/src/main/scala/sbt/serialization/Using.scala

```diff
@@ -0,0 +1,19 @@
+package sbt.serialization
+
+import java.io.{ File, FileOutputStream, BufferedWriter, OutputStreamWriter }
+import java.nio.charset.Charset
+
+object Using {
```

Comment:
Yes this should be forever private. In fact `UFT8FileOutput` should also be private, and eventually be replaced by better implementation from Scala Pickling itself. I opened scala/pickling#309 for it.